bitnami/redis/values.yaml

```diff
@@ -1082,7 +1082,7 @@ sentinel:
   externalMaster:
     enabled: false
     host: ""
-    port: 6379
+    port: 26379
```

Comment:
Is there any reason to change the default value for `externalMaster.port`?

Reply:
Because it block of sentinel